build-and-deploy.sh

```diff
@@ -105,9 +111,13 @@ main() {
   buildPush "backend" "inspirehep/hep"
   maybeBuildSmokeTests
   logout
-  deployQA "ui"
-  deployQA "hep"
-  sentryQA
-  maybeDeploySmokeTestsQA
+  if [ -z "${TRAVIS_TAG}" ]; then
+    deployQA "ui"
+    deployQA "hep"
+    sentryQA
+    maybeDeploySmokeTestsQA
+  else
+    sentryPROD
```

Comment:
question @benjamin-bergia @drjova wouldn't it be convenient to also change the tags in the prod environment automatically through a trigger? given we're using a prod branch anyway for the kubernetes repo, there is no danger of things getting auto-deployed to prod and would simplify things as you don't have to actively watch the image creation anymore.

Reply:
That's a good point, actually. 👍

Reply:
I added a commit to do that @drjova